src/test/java/com/telefonica/iot/cygnus/interceptors/GroupingRuleTest.java

```diff
@@ -66,4 +68,82 @@ public void testFiwareServicePathStartsWithSlash() {
         } // try catch
     } // testFiwareServicePathStartsWithSlash
     
+    @Test
+    public void testIfGetsAreNotNull() {
+        System.out.println(getTestTraceHead("[GroupingRule.groupingRule]")
```

Comment:
The idea is to put here `[class_name.method_tested]`. Nevertheless, in this case several `getXXX` methods are tested... so I would write `[GroupingRule.getXXX]`.